sdk/src/wallet/operations/wait_for_tx_acceptance.rs

```diff
@@ -62,14 +63,10 @@ where
             .block_id
             .ok_or_else(|| ClientError::TransactionAcceptance(transaction_id.to_string()))?;
 
-        let duration = std::time::Duration::from_millis(interval.unwrap_or(DEFAULT_AWAIT_TX_ACCEPTANCE_INTERVAL));
-        for _ in 0..max_attempts.unwrap_or(DEFAULT_AWAIT_TX_ACCEPTANCE_MAX_AMOUNT) {
-            #[cfg(target_family = "wasm")]
-            gloo_timers::future::TimeoutFuture::new(duration.as_millis() as u32).await;
-
-            #[cfg(not(target_family = "wasm"))]
-            tokio::time::sleep(duration).await;
-
+        let duration = interval
+            .map(std::time::Duration::from_millis)
+            .unwrap_or(DEFAULT_WAIT_FOR_TX_ACCEPTANCE_INTERVAL);
+        for _ in 0..max_attempts.unwrap_or(DEFAULT_WAIT_FOR_TX_ACCEPTANCE_MAX_AMOUNT) {
```

Comment:
not 100% sure but I think this might do the option check every iteration for no reason